content/5.resources/1.networks.md

```diff
@@ -18,29 +18,31 @@ Coming Soon
 
 Stable testing network for dapp developers building dApps.
 
-- Chain ID: `constantine-2`
+- Chain ID: **constantine-2**
 - RPC: https://rpc.constantine-2.archway.tech
-- GRPC: `grpc.constantine-2.archway.tech:443`
+- GRPC: **grpc.constantine-2.archway.tech:443**
 - Faucet: https://faucet.constantine-2.archway.tech
 - LCD: https://api.constantine-2.archway.tech
 - Block Explorer: https://explorer.constantine-2.archway.tech
-- P2P Seed: `e5e4a459b85e7bd3f1769e08b7f488b9d0648844@104.154.204.236:26657`
+- Genesis: https://github.com/archway-network/networks/tree/main/constantine-2
+- Persistent Peers: https://github.com/archway-network/networks/blob/main/constantine-2/chain.json#L54
+- P2P Seed: *e5e4a459b85e7bd3f1769e08b7f488b9d0648844@104.154.204.236:26657*
```

Comment:
I think we need to be consistent and either bold everything or italicize everything. I think in this case using the code element is best.

Reply:
ok, i reverted back the styling